assets/javascripts/omise-myaccount-card-handler.js

```diff
@@ -85,11 +86,10 @@
 				Omise.setPublicKey(omise_params.key);
 				Omise.createToken("card", card, function (statusCode, response) {
 				    if (statusCode == 200) {
-				    	$( '#omise_card_name' ).val("");
-				    	$( '#omise_card_number' ).val("");
-				    	$( '#omise_card_expiration_month' ).val("");
-				    	$( '#omise_card_expiration_year' ).val("");
-				    	$( '#omise_card_security_code' ).val("");
+						$.each( omise_card_fields, function( index, field ) {
+							field.val( '' );
+						} );
+
 				    	data = {
 								action: "omise_create_card", 
```

Comment:
is that code formatting correct? regarding indents.

Reply:
the new one is correct, the old one is wrong. 
But good catch 👍  I'm afraid to re-adjust the whole file cuz it's going to be super hard for you guys to review, also I was thinking that this file will be modified more when I work on the API translation stuff. So it's kinda mixed style now.